CoalitionScout.java

```diff
@@ -87,7 +87,9 @@ public void leakDocuments(){
             if (partners != null) {
                 System.out.println(lead.getName() + " wilt coalitieverkenning met: ");
                 for (Party partner : partners) {
-                    System.out.println(partner.getName() + " ");
```

Comment:
De uitwerking is goed. 

Door `println` te gebruiken wordt alles op een nieuwe regel geprint. Je zou ook eerst per mogelijke coalitie een `string` kunnen opbouwen (wellicht met `StringBuilder` zoals vorige keer bekeken) en pas op het einde deze te printen naar de `System.out` .